src/test/java/org/folio/list/service/ListServiceTest.java

```diff
@@ -173,9 +172,32 @@ void testCreateList() {
     when(entityTypeClient.getEntityType(entity.getEntityTypeId())).thenReturn(entityType);
     when(fqlService.getFql(entity.getFqlQuery())).thenReturn(new Fql(equalsCondition));
     when(userFriendlyQueryService.getUserFriendlyQuery(equalsCondition, entityType)).thenReturn(userFriendlyQuery);
-    when(listMapper.toListDTO(entity)).thenReturn(expected);
 
     var actual = listService.createList(listRequestDto);
+    ListDTO expected = new ListDTO()
+      .id(actual.getId())
+      .name(listRequestDto.getName())
+      .description(listRequestDto.getDescription())
+      .entityTypeId(listRequestDto.getEntityTypeId())
+      .entityTypeName(null)
+      .userFriendlyQuery(userFriendlyQuery)
+      .fqlQuery(listRequestDto.getFqlQuery())
+      .fields(listRequestDto.getFields())
+      .createdBy(actual.getCreatedBy())
+      .createdByUsername("SYSTEM")
+      .createdDate(actual.getCreatedDate())
```

Comment:
There were a few properties here that were easier to just pull off `actual`, but those are mostly pretty unimportant or just aren't really available elsewhere